### PR TITLE
#63 UI Overview documentation for v1.1

### PR DESCRIPTION
**Pull request by enriquemeseguerinbibo**
### Description of the issue 
This issue updates the UI Overview page and other UI-related changes for the documentation of version 1.1

### Tasks
 - Update UI overview
   - Maya Shelf
     - Add entries for AdnSkinMerge and Slide On Segment.
     - Update Menu Shortcuts.
   - Maya Menu
     - Update all screenshots.
     - Add sections for Deformers, Solvers, Edit Deformers and Debug submenus.
     - Add License and Help sections.
 - Update all Maya screenshots from A Simple Setup with the most recent shelf.

### CODE REVIEW

File: docs/maya/ui_overview.md [hunk 1]
```diff
@@ -81,6 +85,22 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
...
+In the Deformers section, there is a menu item to open the UI to create an AdnSkinMerge node.
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Solvers](images/adonis_menu_deformers.png)
+  <figcaption><b>Figure 3</b>: AdonisFX Menu Create Deformers section.</figcaption>
+</figure>
+
+#### Solvers
+
+In the Deformers section, there are all options to create AdonisFX simulation deformer nodes.
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:97]:
```suggestion
In the Solvers section, there are all options to create AdonisFX simulation deformer nodes.
```

File: docs/maya/ui_overview.md [hunk 2]
```diff
@@ -81,6 +85,22 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
 
 #### Deformers
 
+In the Deformers section, there is a menu item to open the UI to create an AdnSkinMerge node.
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Solvers](images/adonis_menu_deformers.png)
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:91]:
```suggestion
  ![AdonisFX Menu Create Deformers](images/adonis_menu_deformers.png)
```

File: docs/maya/ui_overview.md [hunk 3]
```diff
@@ -81,6 +85,22 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
 
 #### Deformers
 
+In the Deformers section, there is a menu item to open the UI to create an AdnSkinMerge node.
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:88]: Should we generalize the message and not focus on the `AdnSkinMerge`?
```suggestion
In the Deformers section, there is a menu item to open AdnSkinMerge node creation UI.
```

File: docs/maya/ui_overview.md [hunk 4]
```diff
@@ -81,6 +85,22 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
...
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Solvers](images/adonis_menu_deformers.png)
+  <figcaption><b>Figure 3</b>: AdonisFX Menu Create Deformers section.</figcaption>
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:92]: Check figure number for the entire file.

File: docs/maya/ui_overview.md [hunk 5]
```diff
@@ -136,9 +165,28 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
 - **Draw Fibers**. Enables *Muscle Fibers* Debugging from all the AdnRibbonMuscle and AdnMuscle in the scene.
 - **Hide Fibers**. Disables *Muscle Fibers* Debugging from all the AdnRibbonMuscle and AdnMuscle in the scene.
 
+#### Debug
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Solvers](images/adonis_menu_debug.png)
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:171]:
```suggestion
  ![AdonisFX Menu Debug Section](images/adonis_menu_debug.png)
```

File: docs/maya/ui_overview.md [hunk 6]
```diff
@@ -136,9 +165,28 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
...
 ### Tools section
 
 - **Import**. Launches the AdonisFX Asset Definition Importer to load and create AdonisFX nodes from file (.JSON).
 - **Export**. Launches the AdonisFX Asset Definition Exporter to save AdonisFX nodes information into a file (.JSON).
 - **Paint Tool**. Opens the Paint Tool UI to modify the paintable maps in AdnSkin, AdnMuscle and AdnRibbonMuscle deformers.
 - **Interactive Playback**. Enables Maya Interactive Playback. In this playback mode, dynamic interaction with objects and parameters while simulating is allowed.
+
+### License section
+
+- **Activate License**. Launches a dialog to activate an AdonisFX license or the trial period if the license hasn't been activated.
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:186]:
```suggestion
- **Activate License**. Launches a dialog to activate an AdonisFX license or the trial period if the license is not activated.
```

File: docs/maya/ui_overview.md [hunk 7]
```diff
@@ -22,17 +22,21 @@ The AdonisFX Shelf can be found in the Maya shelf tab under the label *AdonisFX*
...
 | ![Add Targets](images/adn_add_target.png) | Assigns target objects to the selected deformer (utility compatible with AdnRibbonMuscle and AdnMuscle deformers). The mesh with the deformer applied must be the last element in the selection. | AdonisFX Menu > Edit Muscle > *Add Targets* |
 | ![Remove Targets](images/adn_remove_target.png) | Removes target objects assigned to the selected deformer (utility compatible with AdnRibbonMuscle and AdnMuscle deformers). The mesh with the deformer applied must be the last element in the selection. | AdonisFX Menu > Edit Muscle > *Remove Targets* |
+| ![Add Slide On Segment](images/adn_add_sliding_constraint.png) | Adds slide segments based on the two (or more) transform objects and the mesh with the deformer node applied (utility compatible with AdnRibbonMuscle and AdnMuscle deformers). This mesh must be the last element in the selection. | AdonisFX Menu > Edit Muscle > *Add Slide On Segment* |
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:38]:
```suggestion
| ![Add Slide On Segment](images/adn_add_sliding_constraint.png) | Adds slide segments based on the two (or more) transform objects and the mesh with the deformer node applied (utility compatible with AdnRibbonMuscle and AdnMuscle deformers). The mesh must be the last element in the selection. | AdonisFX Menu > Edit Muscle > *Add Slide On Segment* |
```

File: docs/maya/ui_overview.md [hunk 8]
```diff
@@ -99,6 +119,15 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
...
 - **Connection Editor**. Launches the Sensors Connection Editor UI. This tool will assist with the connectivity of the locators and sensors output plugs with the AdonisFX deformer nodes.
 
+#### Deformers
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Solvers](images/adonis_menu_edit_deformers.png)
+  <figcaption><b>Figure 3</b>: AdonisFX Menu Edit Deformers section.</figcaption>
+</figure>
+
+- **Edit Skin Merge**. Launches the Edit Skin Merge UI. This tool will assist with the edition of simulation and animation meshes lists. To open this UI a mesh with the AdnSkinMerge must be selected.
```
Comment (luisalbaceteinbibo) [inline docs/maya/ui_overview.md:129]:
```suggestion
- **Edit Skin Merge**. Launches the Edit Skin Merge UI. This tool will assist with the edition of simulation and animation meshes lists of an AdnSkinMerge deformer. To open this UI a mesh with the AdnSkinMerge must be selected.
```

**Review by luisalbaceteinbibo — CHANGES_REQUESTED** [review]

File: docs/maya/ui_overview.md (hunk 8 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md (hunk 7 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md (hunk 6 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md (hunk 5 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md (hunk 3 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done

File: docs/maya/ui_overview.md (hunk 2 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md (hunk 1 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

**Review by luisalbaceteinbibo — APPROVED** [review]

File: docs/maya/ui_overview.md [hunk 9]
```diff
@@ -76,16 +80,32 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
 
 <figure style="width: 50%;" markdown>
   ![AdonisFX Menu Edit AdnSimshape](images/adonis_menu_nodes.png)
-  <figcaption><b>Figure 8</b>: AdonisFX Menu Create Nodes section.</figcaption>
+  <figcaption><b>Figure 6</b>: AdonisFX Menu Create Nodes section.</figcaption>
 </figure>
 
 #### Deformers
 
+In the Deformers section, there is a menu item to open AdnSkinMerge node creation UI.
```
[the remark this answers is not in the document]
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md [hunk 10]
```diff
@@ -76,16 +80,32 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
...
+
+<figure style="width: 50%;" markdown>
+  ![AdonisFX Menu Create Deformers](images/adonis_menu_deformers.png)
+  <figcaption><b>Figure 7</b>: AdonisFX Menu Create Deformers section.</figcaption>
+</figure>
+
+#### Solvers
+
+In the Solvers section, there are all options to create AdonisFX simulation deformer nodes.
```
[the remark this answers is not in the document]
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md [hunk 11]
```diff
@@ -136,9 +165,28 @@ Option boxes can be used to launch a UI to assign the name to the deformer and c
...
 ### Tools section
 
 - **Import**. Launches the AdonisFX Asset Definition Importer to load and create AdonisFX nodes from file (.JSON).
 - **Export**. Launches the AdonisFX Asset Definition Exporter to save AdonisFX nodes information into a file (.JSON).
 - **Paint Tool**. Opens the Paint Tool UI to modify the paintable maps in AdnSkin, AdnMuscle and AdnRibbonMuscle deformers.
 - **Interactive Playback**. Enables Maya Interactive Playback. In this playback mode, dynamic interaction with objects and parameters while simulating is allowed.
+
+### License section
+
+- **Activate License**. Launches a dialog to activate an AdonisFX license or the trial period if the license is not activated.
```
[the remark this answers is not in the document]
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done!

File: docs/maya/ui_overview.md [hunk 12]
```diff
@@ -50,42 +54,70 @@ The AdonisFX Menu provides access to the options in the shelf and other more adv
...
-<figure style="width: 49%;float: left" markdown>
+<figure style="width: 50%;" markdown>
   ![AdonisFX Menu Create Locators](images/adonis_menu_locators.png)
   <figcaption><b>Figure 3</b>: AdonisFX Menu Create Locators section.</figcaption>
 </figure>
 
-<figure style="width: 49%;float: right" markdown>
+- **Position**. Creates an AdnLocatorPosition node.
+- **Distance**. Creates an AdnLocatorDistance node.
+- **Rotation**. Creates an AdnLocatorRotation node.
```
Comment (carlosmonteagudoinbibo) [inline docs/maya/ui_overview.md:66]: Can we reuse the descriptions used in the shelf icons? Also, I would add a cross reference to the icon saying something like "This item corresponds to the shelf button XXX.png"

**Review by carlosmonteagudoinbibo — CHANGES_REQUESTED** [review]

File: docs/maya/ui_overview.md (hunk 12 above)
Comment (enriquemeseguerinbibo) [reply docs/maya/ui_overview.md]: Done! Applied the additional comment to all menu items that appear on the shelf.

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**